java/client/src/org/openqa/selenium/PrintsPage.java

```diff
@@ -0,0 +1,21 @@
+// Licensed to the Software Freedom Conservancy (SFC) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The SFC licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+package org.openqa.selenium;
+
+public interface PrintsPage {
+  <X> X print(Object printOptions) throws WebDriverException;
```

Comment:
`Pdf print(PrintOptions options)`

Just returning `X` doesn't make it easy to use this API, since users don't know what `X` may be: once type erasure is applied, this ends up as `Object`. I'm not sure why you've created a `PrintOptions` class only to not use it here --- this seems like the ideal place, no?